validator/client/src/main/java/tech/pegasys/teku/validator/client/PendingDuties.java

```diff
@@ -116,6 +116,11 @@ public synchronized void recalculate() {
         });
   }
 
+  @SuppressWarnings("unchecked")
+  public SafeFuture<Optional<T>> getScheduledDuties() {
+    return (SafeFuture<Optional<T>>) duties;
```

Comment:
given we're in this class, i'm super tempted to rename `duties` -> `scheduledDutiesFuture`, might be worth seeing what it'd look like

Reply:
i'd probably also make this do the work...
```
public Optional<T> getScheduledDutues() {
    return duties
          .filter(SafeFuture::isCompletedNormally)
          .flatMap(SafeFuture::getImmediately);
}
```
just returning empty however that works, i wont attempt to do that without playing :)